### ISSUE TITLE
About CentOs

### ISSUE THREAD

**Issue by 111Moderato** [opencv/dldt#343]
In the documentation, I noticed:

> NOTE: Since the TensorFlow framework is not officially supported on CentOS*, the Model Optimizer for TensorFlow can't be configured and ran on those systems.

Can't I use vino to run Tensorflow model in CentOS at all?

**Comment by asuhov** [comment]
Hello, 
You can run Inference Engine with pre-generated IR on CentOS 7.
System requirements for inference published here: https://software.intel.com/en-us/articles/OpenVINO-RelNotes#inpage-nav-12-2

**Comment by 111Moderato** [comment]
I created a simple tensorflow model based on MNIST，and then transformed it with optimizer。But it would cause error about "dropout". If I remove the dropout layer in the trainning phase ,the optimizer will success.
What can i do if I want remain the "dropout"  layer in training phase?  
PS:
" --offload_unsupported_operations_to_tf " doesn't work.  it shows:
`mo_tf.py: error: unrecognized arguments: --offload_unsupported_operations_to_tf`
